ProcessList.h

```diff
@@ -97,4 +99,6 @@ Process* ProcessList_getProcess(ProcessList* this, pid_t pid, bool* preExisting,
 
 void ProcessList_scan(ProcessList* this);
 
+extern long _scanTs;
```

Comment:
Please don't use a leading underscore (maybe `ProcessList_scanTs`?)

Reply:
> Please don't use a leading underscore (maybe `ProcessList_scanTs`?)

Ack, will change.

> Probably the type should also be y2038 safe on 32-bit architectures -> time_t/uint64_t?

I think any of these types will be fine. I used long because we assign it to timespec.tv_sec which is a long.